packages/docs/docs/whisper-wasm/download-whisper-model.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: downloadWhisperModel()
+crumb: '@remotion/whisper-wasm'
+---
+
+# downloadWhisperModel()
+
+Downloads a Whisper.cpp model to the browser.
+
+```tsx twoslash title="app.tsx"
+import {useEffect, useState} from 'react';
+import {downloadWhisperModel} from '@remotion/whisper-wasm';
+
+export default function MyApp() {
+  const [progress, setProgress] = useState(0);
+  useEffect(() => {
+    downloadWhisperModel({model: 'tiny.en', onProgress: setProgress});
+  }, []);
+
+  return <h1>my great app</h1>;
+}
+```
+
+## Options
+
+### `model`
+
+The model to download. Possible values: `tiny`, `tiny.en`, `base`, `base.en`, `small`, `small.en`, `medium`, `medium.en`, `large-v1`, `large-v2`, `large-v3`, `large-v3-turbo`.
```

Comment:
I think ggerganov wrote that anything bigger than `small` does not work well in the browser